crest/Assets/Crest/Crest/Shaders/Resources/UpdateFoam.compute

```diff
@@ -37,20 +39,17 @@ void UpdateFoam(uint3 id : SV_DispatchThreadID)
 	half foam = 0.0;
 	const half r_max = 0.5 - _LD_Params_Source[sliceIndexSource].w;
 
+	// Try to sample the source slice, if it the sample position lies within its footprint?
```

Comment:
`// Try to sample the source slice, if the sample position lies within its footprint`

(minor grammar fix and that is what is happening, so remove question mark)